README.md

```diff
@@ -1,44 +1,44 @@
+## Why RuSTy?
 
-[Structured text](https://en.wikipedia.org/wiki/Structured_text) compiler written in Rust
+Structured Text is a popular language in the domain of automation. A standardized specification of the language ([IEC 61131](https://en.wikipedia.org/wiki/IEC_61131)) was published in the 90s. It was updated several times in the meantime, while its initial spirit - being built for cyclic, robust and deterministic automation applications - still applies.
 
-## About RuSTy
+Several automation platform suppliers built proprietary compilers and runtime libraries, native to the vendor's hard- and software platform.
 
-RuSTy is a structured text (ST) compiler written in Rust. RuSTy utilizes the
-LLVM framework to compile eventually to native code.
+RuSTy is aiming towards a **fast**, **modern** and **open-source** industry-grade ST compiler for a wide range of platforms, sticking close to the standard.
 
 ## Getting started
 
 The easiest way to compile this project is to use the provided `Dockerfile`. The project offers a `.devcontainer` when using [VSCode](https://code.visualstudio.com/docs/remote/containers). The Dockerfile offers a linux-image which contains everything you need to run `cargo build` / `cargo test` in the project's root directory.
 
 If you want to build the project without docker, start [here](https://plc-lang.github.io/rusty/build_and_install.html).
 
-### Documentation
-
-The compiler's documentation can be found here: [documentation](https://plc-lang.github.io/rusty/).
 
-### Contributing
 
-If you want to contribute to the project you should look for some [beginner-friendly issues](https://github.com/PLC-lang/rusty/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and reach out to project's maintainers.
-
-## Why RuSTy
-
-Structured Text is a popular language in the domain of automation. A standardized specification of the language ([IEC 61131](https://en.wikipedia.org/wiki/IEC_61131)) was published in the 90s. It was updated several times in the meantime, while its initial spirit - being built for cyclic, robust and deterministic automation applications - still applies.
-
-Several automation platform suppliers built proprietary compilers and runtime libraries, native to the vendor's hard- and software platform.
-
-RuSTy is aiming towards a _fast_, _modern_ and _open-source_ industry-grade ST compiler for a wide range of platforms, sticking close to the standard.
-
-## Dependencies
+<!-- ## Dependencies
 
 We use the [_logos_](https://crates.io/crates/logos/)
 crate library to perform lexical analysis before a handwritten recursive decent parser creates the AST.
-Generating LLVM IR is accomplished with the help of [_inkwell_](https://github.com/TheDan64/inkwell), a Rust-wrapper around the native LLVM C-API.
+Generating LLVM IR is accomplished with the help of [_inkwell_](https://github.com/TheDan64/inkwell), a Rust-wrapper around the native LLVM C-API. -->
+
+<!-- This seems to be broken?
```

Comment:
tokei breaks often, but it also works enough times..
Why are the other ones disabled?

Reply:
Except for the tokei and codecov badge they felt redundant since the last commit on the master branch already provides GitHub actions information. Do you want me to re-enable them?

Reply:
Done, rendered https://github.com/PLC-lang/rusty/tree/volsa/readme?tab=readme-ov-file#rusty